docs/interfaces_and_networks.md

```diff
@@ -70,6 +70,8 @@ spec:
         networkName: ovs-br1
 ```
 
+> **Note**: It's not recommended to use CNI plugin [macvlan](https://www.cni.dev/plugins/current/main/macvlan/) with bridge interface, since the unicast frame to VM will be dropped without `passthru` mode.
```

Comment:
"Not recommended" is not strong enough if macvlan just won't work with bridge mode